GitExtUtils/GitUI/ThreadHelper.cs

```diff
@@ -96,5 +94,11 @@ public static T CompletedResult<T>(this Task<T> task)
             return task.Result;
 #pragma warning restore VSTHRD002 // Avoid problematic synchronous waits
         }
+
+        public static void RunAsyncAndForget(Func<Task> asyncAction)
```

Comment:
VS threading libraries (where most of this plumbing originated from) use `FileAndForget`. Now we have two different methods - `FileAndForget` and `RunAsyncAndForget`, which is not very good.
Please keep the naming of `FileAndForget`.